packages/core/admin/server/src/controllers/admin.ts

```diff
@@ -185,4 +186,46 @@ export default {
 
     return data;
   },
+
+  async getGuidedTourMeta(ctx: Context) {
+    const userService = getService('user');
+    const { id } = ctx.query as GetGuidedTourMeta.Request['query'];
+
+    const isFirstSuperAdminUser = id ? await userService.isFirstSuperAdminUser(id) : false;
+
+    // Check if any content-type schemas have been create on the api:: namespace
```

Comment:
i think most of this controller logic belongs in a service

Reply:
I considered it and I will create the service to follow convention but just to share my thinking:
- Seemed like overkill to create a service to have one function for this logic
- The service doesn't have much benefit beyond scoping/isolating the code. We won't write a unit test for it since we have the api test, and we don't need to expose it on the strapi instance.